src/Sulu/Bundle/AdminBundle/Resources/js/views/Tabs/Tabs.js

```diff
@@ -129,41 +131,65 @@ class Tabs<T> extends React.Component<Props<T>> {
                 ? this.sortedTabRoutes.findIndex((childRoute) => childRoute === childComponent.props.route)
                 : undefined;
 
+        const disableGap = selectedIndex !== undefined
+            ? this.sortedTabRoutes[selectedIndex]?.options?.tabGap === false
+            : false;
+
+        const showTabs = isRootView || this.sortedTabRoutes.length > 1;
+        const tabType = isRootView ? 'root' : 'nested';
+
+        const className = classNames(
+            tabsStyles.tabsContainer,
+            tabsStyles[tabType],
+            {
+                [tabsStyles.disableGap]: disableGap,
+            }
+        );
+
         return (
             <Fragment>
-                <div className={tabsStyles.tabsContainer}>
-                    <TabsComponent onSelect={this.handleSelect} selectedIndex={selectedTabIndex}>
-                        {this.sortedTabRoutes.map((tabRoute) => {
-                            const tabTitle = tabRoute.options.tabTitle;
-                            const tabBadges = tabRoute.options.tabBadges || [];
-
-                            const badges = (Object.values(tabBadges): any).map((badge: BadgeOptions, index: number) => {
-                                if (typeof badge !== 'object') {
-                                    throw new Error(
-                                        `The value of a badge entry must be an object, but ${typeof badge} was given!`
-                                    );
-                                }
+                {title && <h1>{title}</h1>}
```

Comment:
no sure if this is used anywhere?

Reply:
We need this for the new `Activity` view, otherwise we cannot show a title between the first and the second tab bar